python/test/annotator/embeddings/open_ai_embeddings_test.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from sparknlp.annotator import *
+from sparknlp.base import *
+from pyspark.sql import DataFrame
+from pyspark.sql import SparkSession
+
+class OpenAIEmbeddingsTestCase(unittest.TestCase):
```

Comment:
Please add `@pytest.mark.slow` annotation on top of class definition.

Reply:
Hi @danilojsl,
 I have added the `@pytest.mark.slow` annotation with my new commit and updated the documentation. 
 https://github.com/JohnSnowLabs/spark-nlp/pull/14334/commits/e4a7c1a43071bec2ea2f7f046baadd3e819711c0